membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -164,12 +164,12 @@ object AccountDetails {
             "end" -> endDate,
             "nextPaymentPrice" -> paymentDetails.nextPaymentPrice,
             "nextPaymentDate" -> paymentDetails.nextPaymentDate,
+            "potentialCancellationDate" -> paymentDetails.nextInvoiceDate,
             "lastPaymentDate" -> paymentDetails.lastPaymentDate,
             "chargedThroughDate" -> paymentDetails.chargedThroughDate,
             "renewalDate" -> paymentDetails.termEndDate,
             "anniversaryDate" -> anniversary(startDate),
             "cancelledAt" -> paymentDetails.pendingCancellation,
-            "subscriberId" -> paymentDetails.subscriberId, // TODO remove once nothing is using this key (same time as removing old deprecated endpoints)
```

Comment:
I removed this while. I was in, as it's not used by manage and was confusing things
https://github.com/guardian/manage-frontend/blob/98923b8d2735de2bd2b4343b5b7701852118c1c6/shared/productResponse.ts#L168
cc @rBangay

Reply:
nice one, I'll remove the old references to this in manage as well 👍

Reply:
https://github.com/guardian/manage-frontend/pull/1346